docs/wiki/protocol/overview.md

```diff
@@ -0,0 +1,74 @@
+# Ethereum Overview
+
+## Network coming online and peer-to-peer computing
+
+The concept of peer-to-peer (P2P) networks, where computers communicate and share resources directly without a central authority, has a rich history dating back to the 1960s. Pioneering projects like the ARPANET project, switched on in late October 1969 became the first wide-area packet-switched network, laid the groundwork for this decentralized approach. ARPANET demonstrated the model of distributed communication, where "every participating node could request and serve content".
+
+![ARPANET-map](img/overview/ARPANET-map.jpg)
+> The map of ARPANET in the 1960s
+
+This decentralized philosophy gained significant traction in the late 1990s with the rise of file-sharing platforms like [Napster](https://en.wikipedia.org/wiki/Napster) (1999). These platforms challenged the traditional client-server model, where users relied on a central server to access data. Napster, and subsequent platforms like [BitTorrent](https://en.wikipedia.org/wiki/BitTorrent) (2001), sparked new conversations and philosophies highlighting the potential for a more democratic and distributed online interaction.
```

Comment:
It's bit of jump from arpanet to torrents :) To bridge this a little:

I would emphasize that internet predecessor, arpanet, was basically a decentralized network because of its simple structure. Later as internet evolves, and with the WWW, the client-server model becomes dominant. P2P networks are gaining traction for distributed technologies..

Reply:
Will add this. I was worried if too much history was being covered so I cut it short.